examples_tests/49.ComputeFFT/remove_padding.comp

```diff
@@ -0,0 +1,44 @@
+// Copyright (C) 2018-2020 - DevSH Graphics Programming Sp. z O.O.
```

Comment:
as per, https://github.com/Devsh-Graphics-Programming/Nabla/pull/63/files#r572210425

I don't think you need this shader

Reply:
this file should be gone

Reply:
Yea I know. I haven't started on that yet
some shaders need to get merged

Reply:
also please give me access to update the PR with the latest commits @devshgraphicsprogramming

Reply:
> weirdly I don't see my recent commits here. but It should appear

you didn't push probably?

No magical settings have changed.

Reply:
No, It was either me being stupid or the GitHub website having a bad interface